packages/itwin/ec3-widget/src/components/EC3Widget.tsx

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useActiveIModelConnection } from "@itwin/appui-react";
+import React, { useCallback, useState } from "react";
+import type { EC3ConfigPropsWithGetEC3AccessToken, EC3ConfigPropsWithRedirectUri } from "./EC3/EC3Config";
+import { EC3Context } from "./EC3Context";
+import { EC3Header } from "./EC3Header";
+import { EC3Router } from "./EC3Router";
+import type { Configuration } from "./EC3/Template";
+import "./EC3Widget.scss";
+
+export type EC3WidgetProps = Omit<EC3ConfigPropsWithRedirectUri, "iTwinId"> | Omit<EC3ConfigPropsWithGetEC3AccessToken, "iTwinId">;
```

Comment:
Maybe `EC3ConfigProps` type could be reused here?

Reply:
`Omit` isn't distributive over unions. I have to do them separately.